if-cycles/com/statements/examples/IfCyclesExample1.java

```diff
@@ -7,44 +7,52 @@
 public class IfCyclesExample1 {
 
 	public static void main(String[] args) throws IOException {
-		System.out.println("Search the score on a scale ECTS!");
-		ScaleECTS();
+		String descriptionprogram = "Search the score on a scale ECTS!";
+		System.out.println(descriptionprogram);
 
-		System.out.print("Do you really want to restart? Y/N");
-		BufferedReader yesnot = new BufferedReader(new InputStreamReader(System.in));
-		String beornottobe = (yesnot.readLine().toString());
+		BufferedReader inputStreamReader = new BufferedReader(new InputStreamReader(System.in));
+		String markValue = null;
+		String decision = null;
+		do {
+			System.out.println("Insert your mark:");
+			markValue = inputStreamReader.readLine().toString();
 
-		if (beornottobe.equals("Y")) {
-			ScaleECTS();
-		} else {
-			System.exit(0);
-		}
+			String ecst = getScaleECTS(markValue);
+			System.out.println("Your ECST is: " + ecst);
+
+			System.out.print("Do you really want to restart? Y/N");
+			decision = inputStreamReader.readLine().toString();
+		} while ("Y".equals(decision));
```

Comment:
I think that need to use upperCase for inputted the text, because someone can try to input "y" without "Y", 
example: "decision.toUpperCase"